### PR TITLE
Fix/torchserve custom

### PR DESCRIPTION
**Pull request by jagadeeshi2i**
Fixes:

- curl request in readme
- canary readme doc

Add:

- yaml for metrics

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: docs/samples/custom/torchserve/docs/canary.md [hunk 1]
```diff
@@ -6,11 +6,12 @@
...
 metadata:
- name: "torchserve-custom"
+  name: "torchserve-custom"
 spec:
- predictor:
-   pytorch:
-     storageUri: "pvc://model-pv-claim"
+  predictor:
+    containers:
+    - image: {username}/torchserve:latest
```
Comment (chauhang) [inline docs/samples/custom/torchserve/docs/canary.md:13]: @jagadeeshi2i Please check, previous version had the storageUri steps

File: docs/samples/custom/torchserve/docs/canary.md [hunk 2]
```diff
@@ -21,12 +22,13 @@ Change the path and deploy
...
+  name: "torchserve-custom"
 spec:
- predictor:
-   canaryTrafficPercent: 20
-   pytorch:
-     storageUri: "pvc://model-store-claim-1"
+  predictor:
+    canaryTrafficPercent: 20
+    containers:
+    - image: {username}/torchserve_v2:latest
```
Comment (chauhang) [inline docs/samples/custom/torchserve/docs/canary.md:30]: @jagadeeshi2i Same as above

File: docs/samples/custom/torchserve/docs/metrics.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+Logout and login as admin with username: admin and password: admin
+
+## For Torchserve
+
+### config.properties
+
+Enable metrics in config.properties
+
+```json
+metrics_address=http://0.0.0.0:8082
```
Comment (chauhang) [inline docs/samples/custom/torchserve/docs/metrics.md:63]: @jagadeeshi2i Shouldn't this be added in the TorchServe dockerfile as well?

File: docs/samples/custom/torchserve/docs/canary.md (hunk 1 above)
Comment (jagadeeshi2i) [reply docs/samples/custom/torchserve/docs/canary.md:13]: We do it by changing the image in custom server without external storage and the storageUri for custom server with external storage.

File: docs/samples/custom/torchserve/docs/metrics.md (hunk 3 above)
Comment (jagadeeshi2i) [reply docs/samples/custom/torchserve/docs/metrics.md:63]: Yes 8082 port is exposed in dockerfile added inside torchserve-image folder.

**Pull request closed without merging**